app/auth/firebase_auth.py

```
@@ -11,7 +11,7 @@
 class FirebaseAuthService:
     def __init__(self):
         self._initialize_firebase()
-        self.jwt_secret = os.getenv("JWT_SECRET", "your-secret-key")
+        self.jwt_secret = "DEMO_HARDCODED_JWT_SECRET_123"
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "jwt_secret"

(S105)